decidim-admin/app/views/decidim/admin/participatory_processes/show.html.erb

```diff
@@ -31,10 +31,14 @@
     :short_description,
     :description
     %>
-  <dt><%= display_label(@participatory_process, :hero_image) %></dt>
-  <dd><%= image_tag @participatory_process.hero_image.url.to_s %></dd>
-  <dt><%= display_label(@participatory_process, :banner_image) %></dt>
-  <dd><%= image_tag @participatory_process.banner_image.url.to_s %></dd>
+  <% if @participatory_process.hero_image %>
+    <dt><%= display_label(@participatory_process, :hero_image) %></dt>
```

Comment:
Why are you hiding the label if the image is not present? Other optional fields keep showing their label when they are not set.